spring-kafka/src/main/java/org/springframework/kafka/config/StreamsBuilderFactoryBean.java

```diff
@@ -225,6 +227,14 @@ public void setCloseTimeout(int closeTimeout) {
 		this.closeTimeout = Duration.ofSeconds(closeTimeout); // NOSONAR (sync)
 	}
 
+	/**
```

Comment:
This needs an `@since` tag. In addition, please add your name as an `author` of this class.